controllers:
  robot_state_controller:
    type: robot_state_controller/RobotStateController
    publish_rate: 50
  imu_sensor_controller:
    type: imu_sensor_controller/ImuSensorController
    publish_rate: 100
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
  chassis_controller:
    type: rm_chassis_controllers/BalanceController
    joint_left_name: "left_wheel_joint"
    joint_right_name: "right_wheel_joint"
    wheel_radius: 0.06
    wheel_base: 0.405
    com_pitch_offset: -0.019
    a: [ [ 0, 1, 0, 0 ],
         [ 21.7204, 0, 0, 0 ],
         [ 12.4948, 0, 0, 0 ],
         [ 0, 0, 0, 0 ] ]
    b: [ [ 0, 0 ],
         [ -64.4355, -64.4355 ],
         [ 7.7211, 7.7211 ],
         [ -7.1659, 7.1659 ] ]
    q: [ [ 0.01, 0, 0, 0 ],
         [ 0, 0.01, 0, 0 ],
         [ 0, 0, 0.2, 0 ],
         [ 0, 0, 0, 0.01 ] ]
    r: [ [ 1, 0 ],
         [ 0, 1 ] ]
    twist_covariance_diagonal: [ 0.001, 0.001, 0.001, 0.001, 0.001, 0.001 ]
